starlite/datastructures/response_containers.py

```diff
@@ -88,22 +92,36 @@ class File(ResponseContainer[FileResponse]):
 
     path: FilePath
     """Path to the file to send"""
-    filename: str
+    filename: Optional[str] = None
     """The filename"""
     stat_result: Optional[os.stat_result] = None
     """File statistics"""
+    chunk_size: int = ONE_MEGA_BYTE
+    """The size of chunks to use when streaming the file"""
+    content_disposition_type: "Literal['attachment', 'inline']" = "attachment"
+    """The type of the 'Content-Disposition'. Either 'inline' or 'attachment'."""
+    etag: Optional[str] = None
+    """An optional etag for the file. If not provided, and etag will be generated automatically."""
```

Comment:
Another typo? `If not provided, an etag`

Reply:
fixed